src/main/javascript/view/partials/signature.js

```diff
@@ -573,7 +573,13 @@ SwaggerUi.partials.signature = (function () {
       value = _.cloneDeep(value);
       var example = value.examples;
       if (_.isString(example)) {
-        example = jsyaml.safeLoad(example);
+        try {
+          example = jsyaml.safeLoad(example);
```

Comment:
@gilbode yep - you're right.  I thought I had done that and was still getting an undefined for jsyaml, but I must have had caching issues... :\